src/modules/pythonmonkey/pythonmonkey.cc

```diff
@@ -318,7 +320,7 @@ static PyObject *eval(PyObject *self, PyObject *args) {
     return NULL;
   }
 
-  if (evalOptions && !PyDict_Check(evalOptions)) {
+  if (evalOptions && !(PyDict_Check(evalOptions) || PyObject_TypeCheck(evalOptions, &JSObjectProxyType))) {
```

Comment:
It's a sub-type, so this is clearly useless

Reply:
https://github.com/Distributive-Network/PythonMonkey/pull/264/commits/59dedb6a36f7585a8eb23d7a9d86a0aa21bd3485